src/views/waypoint/home/components/heading/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { HeadingProps } from './types'
+
+function Heading({
+  __heading_slug,
+  heading,
+  level,
+}: HeadingProps): React.ReactElement {
+  const Component = `h${level}` as React.ElementType
+  return <Component id={__heading_slug}>{heading}</Component>
```

Comment:
nit: Could this just be `id` or `slug`? 💭

Reply:
Nice, yep, definitely could be - updated to `slug` 👍